viam-cartographer.go

```diff
@@ -244,6 +252,14 @@ func (cartoSvc *cartographerService) GetInternalState(ctx context.Context) (func
 	return grpchelper.GetInternalStateCallback(ctx, cartoSvc.Name().ShortName(), cartoSvc.clientAlgo)
 }
 
+// GetLatestMapInfo returns the timestamp stored when GetPointCloudMap was last called.
```

Comment:
[nit] slight more descriptive
```suggestion
// GetLatestMapInfo returns the timestamp associated with the latest call to GetPointCloudMap, unless you are localizing; in which case the timestamp returned is the timestamp of the session.
```

Reply:
Something like that